### ISSUE TITLE
green fringe produced for extra line in rich comment block

### ISSUE THREAD

**Issue by dareno** [clojure-emacs/cider#2937]
## Expected behavior
Evaluating just a single line in a Rich comment block will apply a green fringe to only the line being evaluated.
## Actual behavior
Evaluating just a single line in a Rich comment block will apply a green fringe to the line being evaluated as well as the following line.
## Steps to reproduce the problem
1. add the following comment block to a file:
```
(comment
  (+ 1 1)
  (+ 1 2)
  (+ 1 3))
```
2. evaluate the entire comment block to produce a green fringe just on the first line ("comment" line)
3. evaluate the second commented expression and see that both the second and third expressions will be marked as evaluated (this happens even when following line is blank).

### CIDER version information

```
;; CIDER 1.0.0snapshot (package: 20201125.1146), nREPL 0.8.3
;; Clojure 1.10.1, Java 1.8.0_271
```

### Lein/Boot version
```
Leiningen 2.9.4 on Java 1.8.0_271 Java HotSpot(TM) 64-Bit Server VM
```
### Emacs version
```
GNU Emacs 27.1 (build 1, x86_64-apple-darwin18.7.0, NS appkit-1671.60 Version 10.14.6 (Build 18G95)) of 2020-08-12
```
### Operating system
macOS Catalina Version 10.15.7

**Comment by fvides** [comment]
I just submitted a PR that hopefully fixes this issue.

**Comment by fvides** [comment]
Should someone check this and if everything is okay close it?

**Comment by dareno** [comment]
I just did some manual testing and can no longer reproduce my test case. Looks good to me. Closed by #2937.
